### DISCUSSION TITLE
multiple readers for 'inp' mesh

### DISCUSSION THREAD

**Discussion by jessejcarter** [pyvista/pyvista#4673] [Q&A] [answered]
Hi all, first-time poster here. Excited to get started with PyVista after attending a great tutorial at Scipy 2023!

I'm interested in loading an Abaqus mesh, which typically has an 'inp' file extension. Looks like PyVista uses the `pyvista.AVSucdReader` for reading inp files. However meshio does recognize the Abaqus inp format, but I don't see Abaqus in the list of [supported readers](https://docs.pyvista.org/version/stable/api/readers/index.html).

**Comment by tkoyama010** [comment, marked as answer]
The list is a reader supported by vtk and does not show the `meshio` format. The `meshio` read functionality can be used by explicitly specifying the format in `file_format`. We can also use `read_meshio`.
```python
import pyvista as pv

mesh = pv.read_meshio(filename)
```
